### PR TITLE
Adding validation for Nikname Field and Changing DTOs

### PR DESCRIPTION
**Pull request by PieroRafael**
# Description
Adding validation for Nikname Field and Changing DTOs

## Type of change
- [X] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
Not Yet

### CODE REVIEW

File: bloggio/src/main/java/com/bloggio/api/bloggio/payload/users/ValidatorExistUserNickname.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+@Log4j2
+public class ValidatorExistUserNickname implements ConstraintValidator<ExistUserNickname, String> {
+
+    @Autowired
```
Comment (rricardob) [inline bloggio/src/main/java/com/bloggio/api/bloggio/payload/users/ValidatorExistUserNickname.java:15]: Evitar hacer uso de autowired, hacerlo por constructor

File: bloggio/src/main/resources/application.yml [hunk 2]
```diff
@@ -2,12 +2,12 @@ spring:
...
     name: "bloggio-api"
   datasource:
-    url: jdbc:postgresql://localhost:5432/blog
-    #url: jdbc:postgresql://dpg-cqp4r0dds78s73e4ecs0-a.oregon-postgres.render.com/bloggio_db
-    username: postgres
-    #username: bloggio_db_user
-    password: postgres
-    #password: EtYThWqr3ifUMWjjsh6pYlS7TWT63s7i
+    #url: jdbc:postgresql://localhost:5432/blog
+    url: jdbc:postgresql://dpg-cqp4r0dds78s73e4ecs0-a.oregon-postgres.render.com/bloggio_db
```
Comment (rricardob) [inline bloggio/src/main/resources/application.yml:6]: Los accesos deben ser controlador por variables de entorno

**Review by rricardob — CHANGES_REQUESTED** [review]

File: bloggio/src/main/java/com/bloggio/api/bloggio/payload/users/ValidatorExistUserNickname.java (hunk 1 above)
Comment (PieroRafael) [reply bloggio/src/main/java/com/bloggio/api/bloggio/payload/users/ValidatorExistUserNickname.java:15]: ok jefe , igual no lo voy a hacer

File: bloggio/src/main/resources/application.yml (hunk 2 above)
Comment (PieroRafael) [reply bloggio/src/main/resources/application.yml:6]: mmmmmmmmmmmmmm

**Review by rricardob — APPROVED** [review]

**Pull request merged**